pages/account/statistics.js

```diff
@@ -79,18 +79,45 @@ export async function getServerSideProps(context) {
     profile.links.some((pLink) => pLink.url === link.url)
   );
 
+  const dailyStatsMap = new Map(data.profile.daily.map(stat => [
+    new Date(stat.date).toISOString().split('T')[0], // Convert to "YYYY-MM-DD" format
```

Comment:
Please don't use split on a date, use the date formatter https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#description